src/app/containers/StoryPromo/index.jsx

```diff
@@ -281,6 +281,7 @@ const StoryPromoContainer = ({
 
   return (
     <StoryPromoComponent
+      data-e2e="story-promo"
```

Comment:
This component is a generic one, are we adding it here to identify all instances of the storypromo in e2e's ? is there a case where we would want to distinguish across instances within tests ? if so we might want to have this prop here and let those components pass in a unique value.

Reply:
At the moment, there is no requirement to be able to distinguish between different types, but agree it could be something we do in future, if required. (e.g. we already do this for the CpsOnwardJourney component https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsOnwardJourney/index.jsx#L174)